lectures/01-intro.md

```diff
@@ -53,39 +57,31 @@ title: Scala 2024 (tm)
 
 ::: { .fragment }
 
-* Домашни през семестъра: 50 точки
-* Два теста: 50 точки
+* Домашни през семестъра (около 5): 50 точки
 * Финален проект: 50 точки
 * Бонус точки
-* Общо: 150+
+* Общо: 100+
+* Ще следим за стил (огромна част от оценката)
```

Comment:
Propose to do "important for the overall score" vs "huge part of the score" and when we discuss should be clear on what is meant, otherwise it sounds sus & arbitrary.